tests/fixtures/ember-autofocus-modifier-v5.0.0/package.json

```diff
@@ -59,7 +59,7 @@
     "ember-resolver": "^8.0.2",
     "ember-source": "~4.7.1",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.3.1",
+    "ember-template-lint": "^5.10.3",
```

Comment:
I was getting errors from etl when running tests related to the recents issues (like https://github.com/ember-template-lint/ember-template-lint/issues/2916). Not sure why though, because the fixtures have no lockfile, so should have pulled the latest (in-range) anyway!? 🤔  Maybe again, just some quirks on my local machine?

Btw, making the copy calls include dotfiles might have also fixed some false positives here, as things like https://github.com/NullVoxPopuli/ember-addon-migrator/blob/main/tests/fixtures/ember-autofocus-modifier-v5.0.0/.template-lintrc.js where not getting copied correctly...

Reply:
naw, I am familiar with this! `pnpm` tends to favor what's in your local store when resolving a semver range rather than jumping to latest possible

Reply:
Oh, really? 😱
That sounds wrong tbh!?

Reply:
Yeah, i don't like the behavior